map/src/main/java/openlr/map/utils/GeometryUtils.java

```diff
@@ -316,50 +329,19 @@ private static double hMult(final double y) {
     public static double calculateLineBearing(final Line line,
                                               final BearingDirection dir, final int pointDistance,
                                               final int projectionAlongLine) {
-        if (line == null) {
+        if (line == null || projectionAlongLine < 0 || projectionAlongLine > line.getLineLength()) {
             return -1.0;
         }
-        GeoCoordinates p1 = null;
-        GeoCoordinates p2 = null;
 
+        GeoCoordinates p1 = line.getGeoCoordinateAlongLine(projectionAlongLine);
+        GeoCoordinates p2 = null;

```

Comment:
`null` is redundant

Reply:
0975cd3318040870779bc3de4b706cd2ffb8b2ef